UPGRADE-6.2.md

```diff
@@ -41,6 +41,11 @@ HttpFoundation
  * Deprecate `Request::getContentType()`, use `Request::getContentTypeFormat()` instead
  * Deprecate calling `JsonResponse::setCallback()`, `Response::setExpires/setLastModified/setEtag()`, `MockArraySessionStorage/NativeSessionStorage::setMetadataBag()`, `NativeSessionStorage::setSaveHandler()`   without arguments
 
+HttpClient
+----------
+
+ * Deprecate implementing `RequestFactory`, `StreamFactory` and `UriFactory` on `HttplugClient`
```

Comment:
I would use the FQCN or make it clear that those are HTTPlug interfaces

Reply:
updated, thanks for the review